commands/srandmember.md

```diff
@@ -1,13 +1,15 @@
 When called with just the `key` argument, return a random element from the set value stored at `key`.
 
-Starting from Redis version 2.6, when called with the additional `count` argument, return an array of `count` **distinct elements** if `count` is positive. If called with a negative `count` the behavior changes and the command is allowed to return the **same element multiple times**. In this case the number of returned elements is the absolute value of the specified `count`.
+If the provided `count` argument is positive, return an array of `count` **distinct elements**.
+If called with a negative `count`, the behavior changes and the command is allowed to return the **same element multiple times**. In this case, the number of returned elements is the absolute value of the specified `count`.
 
-When called with just the key argument, the operation is similar to `SPOP`, however while `SPOP` also removes the randomly selected element from the set, `SRANDMEMBER` will just return a random element without altering the original set in any way.
+Conceptually, this command is similar to `SPOP`. However, while `SPOP` also removes the randomly selected element from the set, `SRANDMEMBER` only returns a random element without altering the original set in any way.
```

Comment:
not sure if that line is necessary

Reply:
Removed via ca2bd9a3da2ad5e88dd2ef434cf21e6062b8649f